plugins/woocommerce/tests/php/includes/wc-template-functions-test.php

```
@@ -0,0 +1,188 @@
+		$wp_query->is_page = true;
+		$wp_query->is_404 = false;
+		$wp_query->queried_object_id = $page_id;
+		$wp_query->queried_object = get_post( $page_id );
+		$wp_query->post = get_post( $page_id );
+
+		ob_start();
+		
+		$is_wc_endpoint = is_wc_endpoint_url();
+		$is_order_received = is_order_received_page();
+		$is_checkout_pay = is_checkout_pay_page();
+		$should_trigger_404 = apply_filters( 'woocommerce_account_endpoint_page_not_found', true );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$should_trigger_404'. (undefined)

(UnusedLocalVariable)